.scrutinizer.yml

```diff
@@ -0,0 +1,12 @@
+build:
```

Comment:
I'm in doubt about Scrutinizer, recently seen a bit too much issues on big projects with many dependencies. But we can try it of course to see what it brings us :)

Reply:
It sometimes brings issues that phpstan haven't found. Also I like the code quality badge ;)